README.md

```diff
@@ -1,3 +1,194 @@
 # PatternFly developer training
 
-More information about how to contribute is coming soon.
+The team at PatternFly has created developer training to teach PatternFly implementation best practices. We chose (Katacoda)[https://www.katacoda.com/patternfly] as the platform for delivering the training modules. Check out the (React)[https://www.patternfly.org/v4/documentation/react/overview/training] and (HTML/CSS)[https://www.patternfly.org/v4/documentation/core/overview/training] tutorials on PatternFly, and share your feedback by submitting a review on the final page of each module. You can also file an issue in this repository.
+
+## Submitting a proposal
+
+If you would like to add a new module to the current PatternFly developer training set, thoroughly familiarize yourself with the existing modules in the category you would like to add a module to, so there is no duplicate work created. Once that is accomplished, create a new issue for your proposal in this repository. Our team will review all submissions to make sure that modules align with the goals of training and to avoid overlap of content.
+
+Be sure to cover these points in your submission:
+
+1. Title of module
+2. Goal of the module (what will the user learn)
+3. Verify module builds on the documentation that is already on (PatternFly)[https://www.patternfly.org/v4/].
+4. Description of the module
+5. Assign module to relevant category:
+  1. HTML/CSS: Introductary and high-level concepts
+  2. React: Basic: React principles and customization
+  3. React components: React components including select, table, and toolbar.
+  4. React Charts: Reat chart components including, Area, bar, bullet, donut, donut utilization, line, pie, sparkline and stack charts.
+  5. None of the above. Suggestion for a new category.
+6. Approximate length of the module
+    1. Short ( < 5 mins)
+    2. Medium  ( < 15 mins)
+    3. Long (> 30 mins)
+
+## Setup development environment and plan out your scenario
+
+Once your proposal has been approved by our team you can start developing. We suggest following this order:
+
+1. Write the title, description and sub-steps for each step in a document. This will help to organize your thoughts.
+2. It is not possible to run your code locally when developing in Katacoda so we suggest using CodePen or CodeSandbox to test the working code for your example.
+3. Once you feel confident in your steps and code examples, fork our repository: https://github.com/patternfly/training-scenarios.
+4. Add the new module to the repository.
+5. Push up the addition and submit a Pull Request.
+6. Tag at least two members of the PatternFly team to review your Pull Request.
+To see your changes live on Katacoda before being merged:
+7. Submit a second Pull Request that merges changes from the new branch on your fork, to the master branch on your fork, and merge the changes.
```

Comment:
These should be new steps, starting from 1. The previous set of steps (1 – 6) were dev instructions. Then we say, "To see your changes live," which involve a new set of instructions.